pages/phases/job-announcement/index.html

```diff
@@ -62,19 +62,38 @@ <h4 class="chp-highlight__title">
   </ul>
 </div>
 
-<h2>Final checklist before JOA Posting</h2>
+<h2>Planning checklist before JOA Posting</h2>
 
 <ul>
-<li>If using USA Staffing, have you set up the manual assessments in order to import the SMEQA assessment results into Staffing? <b>Not doing this will cause a large amount of manual work.</b></li>
-<li>Do you have a final private copy of the competencies and proficiencies that SMEs have approved with language matching the JOA?</li>
+  <li>If using USA Staffing, have you set up the manual assessments in order to import the SMEQA assessment results into Staffing? <b>Not doing this will cause a large amount of manual work.</b></li>
+  <li>Have you determined with HR and hiring managers what assessment materials will be available in the selection process (and explained that on the JOA?)</li>
+  <li>Do you have a final private copy of the competencies and proficiencies that SMEs have approved with language matching the JOA?</li>
+  <li>Is your HR specialist in office and ready to process applicants any day the application may close?</li>
+  <li>Do you have a recruitment strategy? (Are the SMEs and HMs ready to do recruitment when you send them the link?)</li>
+</ul>
+
+<h2>Content checklist before JOA Posting</h2>
+<ul>
 <li>Do you have language calling out the page limit for resume review in the JOA</li>
 <li>Have you included language allowing candidates to be considered for other hiring actions?</li>
-<li>Have you included the opt-in questionnaire item on being considered for employment at other agencies?</li>
+<li>Do you explain that each round of assessment is pass/fail?</li>
 <li>Do you have a link to the questionnaire in your JOA?</li>
+<li>Have you included the opt-in questionnaire item on being considered for employment at other agencies?</li>
 <li>Did you consider including language telling applicants the announcement will remain open for either 240 days or 365 days to maximize selections off the certificate throughout the year?</li>
-<li>Are the SMEs and HMs ready to do recruitment when you send them the link?</li>
-<li>Is your HR speicialist in office and ready to process applicants any day the application may close?</li>
+<li>Did you consider planning a conference call or calls to inform possible applicants about the application process, and putting the dial in information in the JOA?</li>
 <li>Is your announcement clear about the locations of each posting and if you can hire people remotely? (candidates may assume positions are remote-friendly)</li>
+<li>Do you have the elements on your announcement so that you can make successful offers? For example:</li>
+  <ul>
+      <li>Relocation incentives</li>
+      <li>Recruitment incentives/superior quals statements (these are not the same)</li>
+      <li>Promotion potential
+      <li>Position sensitivity/clearance information</li>
+      <li>Relocation incentives</li>
```

Comment:
```This is a duplicate list item, remove

Reply:
I cna't tell what this is saying?

Reply:
```suggestion
```

Reply:
Yes, sorry, somehow I botched the removal of a duplicate.